### ISSUE TITLE
latest asp.net dependencies cause the program not starting properly in IIS server without 4.5.2 installed

### ISSUE THREAD

**Issue by zijianhuang** [zijianhuang/TraceHub#6]
TraceHub had been built on .net 4, .net 4.5 and currently 4.5.2. It has been working well on servers without 4.5.2 installed but 4.6.2. However, after updating with latest ASP.NET dependencies through NuGet, the traceHubSlim demo stopped working.

**Comment by zijianhuang** [comment]
Changing the Web.config with:
```xml
  <system.web>
    <compilation debug="false" targetFramework="4.6.2"  optimizeCompilations="true" />
    <httpRuntime targetFramework="4.6.2" />
  </system.web>
```
resolves the problem, so TraceHub built on 4.5.2 could working on server with only 4.6.2 installed. Apparently this is related to a trick that Microsoft has been using to force software vendors to move on.

Anyway, it looks like a good time to build TraceHub on 4.6.2.

**Comment by zijianhuang** [comment]
done
